packages/addons/service/net-snmp/source/system.d/netsnmp.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
```

Comment:
I do not understand: LE would automatically start/stop service.net-snmp.service.
What starts the two system.d units you provide?